frontend/src/views/Game/Ingame.svelte

```diff
@@ -9,10 +9,76 @@
   export let socket: SocketClient
   export let gameData: GameSocketResponse
   export let currentUser: UserResponse
+  enum GameState {
```

Comment:
should the game state perhaps live in the socket?

We could implement a `inGameState` getter which calculates the state on demand.

Reply:
this will instead be refactored and solved in #70